app/models/applicant.rb

```diff
@@ -0,0 +1,18 @@
+require 'date'
```

Comment:
As spoken offline, this require should not necessary as `date` belongs to the ruby core libraries, so should be required already by default.